requirements_dev.txt

```diff
@@ -9,3 +9,16 @@
 # and also the 'dev' list in the extras_require dict
 
 -e .[dev]
+
+pytest~=6.2.4
```

Comment:
please do not add these here. If there is something you need to add, add it to setup.py. This looks like something pyCharm suggested. But we don't code around an IDE, we code around Python. Please remove these. If anything needs to be added for the sake of functionality, please do so in setup.py. But don't do it just to oblige your IDE.